cmsdb/campaigns/run3_2022_preEE_nano_v11/__init__.py

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+
+"""
+Common, analysis independent definition of the 2022 pre-EE data-taking campaign
```

Comment:
Could you add a small reference here to what the EE refers?
Same for the postEE campaign.

Reply:
Can do. Maybe I can also add a link to this [rather entertaining article](https://cms.cern/news/problems-and-solutions-ecal-leak-story)? :P